src/server/script_mgr.cc

```diff
@@ -185,38 +174,49 @@ void ScriptMgr::LatencyCmd(ConnectionContext* cntx) const {
   }
 }
 
-// Search for pragmas in script body
-io::Result<optional<ScriptMgr::ScriptParams>, GenericError> DeduceParams(string_view body) {
-  static const regex kRegex{"^\\s*?--\\s*?pragma\\s*?:(.*)"};
+// Search for flags shebang if present and truncate shebang.
```

Comment:
should not it be the first line in the script or something?

Reply:
Yes, I'll rename it to check for flags shebang. The regex checks for start